src/lib/y2partitioner/widgets/partitions_delete_button.rb

```diff
@@ -0,0 +1,51 @@
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2partitioner/widgets/device_button"
+require "y2partitioner/actions/delete_partitions"
+
+module Y2Partitioner
+  module Widgets
+    # Button for deleting all partitions
+    class PartitionsDeleteButton < DeviceButton
+      def initialize(*args)
+        super
+        textdomain "storage"
+      end
+
+      # @macro seeAbstractWidget
+      def label
+        # TRANSLATORS: label for button to delete all partitions
+        _("Delete All...")
```

Comment:
Hmm, why not "Delete partitions" ? this Delete All, is quite depending on context and in some context it can look strange.

Reply:
Well, I have just implemented it as described in the document. But, anyway, this button only belongs to the "Partitions tab", so the context is always correct. @ancorgs what do you think? Would it better to rename the button as "Delete Partitions" ?

Reply:
It's a list of partitions, so "Delete All" looks fine to me. In this first prototype, we do not longer have a button to delete all partitions in the table of disks, for example. If we reconsider that (I want to send a mail to Ken about it), then we should re-evaluate the label. But so far it looks correct to me.